src/expressions/dataTypes/Value.ts

```diff
@@ -312,3 +312,16 @@ export function stringToSequenceType(input: string): SequenceType {
 			};
 	}
 }
+
+export function stringToSequenceMultiplicity(input: string): SequenceMultiplicity {
+	switch (input) {
```

Comment:
This could potentially be done more nicely, by doing `switch(input[input.length - 1])`, then it also works for entire type strings.

Reply:
Where I use it, it only gives the +, * or ? operators, hence this should be enough.